tennis-scheduler-web-app/src/components/TennisCourtInfo.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import AddTimeslot from "./AddTimeslot";
+import axios from "axios";
+import "../styles/courts.css";
+
+export const TennisCourtInfo = ({ id, image, name, surfaceType, description }) => {
+
+  const [timeslotErrors, setTimeslotErrors] = useState("");
+  const [buttonName, setButtonName] = useState("Add timeslot");
+  const [showAddTimeslot, setAddTimeslot] = useState(false);
+
+  const addTimeslot = async (timeslot) => {
+
+    let Timeslot = {
+      dateStart: `${timeslot.timeslotDate}T${timeslot.startTime}:02.174Z`,
+      dateEnd: `${timeslot.timeslotDate}T${timeslot.endTime}:02.174Z`,
+      personId: 1,
+      courtId: timeslot.id
+    };
+
+    await axios.post("http://localhost:8081/timeslots/", Timeslot).then(
```

Comment:
Extract this in a new file where you store all of your API calls for timeslots, also use base URL from .env file